sycl/test/host-interop-task/interop-task.cpp

```diff
@@ -0,0 +1,192 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %threads_lib -lOpenCL
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+#include <atomic>
+#include <condition_variable>
+#include <future>
+#include <mutex>
+#include <thread>
```

Comment:
Are all of these headers needed here? For example, why you needed 'atomic' or 'mutex' in this test?

Reply:
Oh, sorry, it's copy-paste. Removed.